.github/workflows/build.yml

```
@@ -0,0 +1,315 @@
+---
+name: Build and Release
+
+"on":
+  push:
+    tags:
+      - "v*"
+  workflow_dispatch:
+    branches: [main, staging]
```

actionlint (1.7.7)

expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)